### ISSUE TITLE
Can not install hadolint using scoop on Windows10

### ISSUE THREAD

**Issue by LGDHuaOPER** [hadolint/hadolint#952]
- [ ] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

On Windows, you can use [scoop](https://github.com/lukesampson/scoop) to install hadolint.

```bash
scoop install hadolint
```

### Actual behavior

![image](https://user-images.githubusercontent.com/22699037/232975690-6700191b-726a-4a34-860c-327d1e68732b.png)

### Steps to reproduce the behavior

1.install

```bash
scoop install hadolint
```

**Additional environment details (OS, stack version, etc.)**

Window10

![image](https://user-images.githubusercontent.com/22699037/232976636-806165d8-f346-40bf-b650-553058ba43e0.png)

**Comment by LGDHuaOPER** [comment]
Thanks!
